test/e2e/test_worker_status.py

```diff
@@ -40,3 +42,83 @@ def test_worker_lifecycle_status_is_expected(
             fleet_id=deadline_resources.fleet.id,
             worker_id=function_worker.worker_id,
         )
+
+    def test_linux_worker_restarts_process(
+        self,
+        deadline_resources,
+        deadline_client: DeadlineClient,
+        function_worker: EC2InstanceWorker,
+    ) -> None:
+        # Verifies that Linux Worker service restarts the process when we start/stop worker process
+
+        assert function_worker.worker_id is not None  # To fix linter type mismatch
```

Comment:
to fix?

Reply:
I think this is needed because the linter will complain without it since `worker_id` is `Optional` IIRC

Reply:
As in this line fixes the linter type mismatch 😆